CarService.Api/Services/OrderService.cs

```diff
@@ -63,6 +66,28 @@ public OrderService(UserManager<User> userManager, IUnitOfWorkFactory unitOfWork
             }
         }
 
+        public async Task CreateReviewProposition(string email, ReviewPropositionCreationDto reviewPropositionDto)
+        {
+            var user = await _userManager.FindByEmailAsync(email);
+
+            if (user != null)
```

Comment:
You've already checked it inside the Controller, isn't enough it?

Reply:
if(user!=null) will be deleted from all Services' method in next commit